app/components/message_options_component.html.erb

```diff
@@ -11,6 +11,10 @@
           Stiahnuť ako PDF
         <% end %>
       <% end %>
+      <%= link_to export_message_path(@message), data: { turbo_frame: '_top', turbo: false }, class: 'whitespace-nowrap flex gap-3', role: 'menu-item', tabindex: -1 do %>
+        <%= render Icons::ArrowDownTrayComponent.new(css_classes: "w-5 h-5") %>
```

Comment:
Aby sme tam nemali vela rovnakych ikon tam pokukaj tu https://heroicons.com/outline

Reply:
Btw nevyrabajme komponent per ikonu, to je bad practice.

Reply:
dajme stiahnut ako pdf `document-arrow-down`

Reply:
<img width="1364" alt="image" src="https://github.com/slovensko-digital/govbox-pro/assets/43437253/6f7eb585-4d4b-46ae-8516-734b588854fe">